Source/Scene/Cesium3DTileset.js

```diff
@@ -892,13 +879,8 @@ define([
 
     function raiseLoadProgressEvents(tileset, frameState) {
         var eventsToRaise = tileset._loadProgressEventsToRaise;
-        var length = eventsToRaise.length;
-        for (var i = 0; i < length; ++i) {
-            var numberOfPendingRequests = eventsToRaise[i].numberOfPendingRequests;
-            var numberProcessing = eventsToRaise[i].numberProcessing;
-
-            frameState.afterRender.push(evenMoreComplicated(tileset, numberOfPendingRequests, numberProcessing));
-        }
+        frameState.afterRender.push(evenMoreComplicated(tileset, tileset._statistics.numberOfPendingRequests,
```

Comment:
Now that we aren't using temp variables like `numberOfPendingRequests` and `numberProcessing`, we don't need to use `evenMoreComplicated` anymore for closure purposes. The function there can go inside the `frameState.afterRender.push()`.